### PR TITLE
First draft of a general Roles SOP

### PR DESCRIPTION
**Pull request by matentzn**
This is a first draft of a general roles SOP. We will open it to the rest of OBO Ops once you two (@nataled and @nlharris) agree with the basic phrasing.

### CODE REVIEW

File: docs/roles/OBORoles.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# OBO Foundry Roles
+
+OBO Foundry Roles are designed to manage a single aspect of the day-to-day activities of the OBO Foundry.
```
Comment (nlharris) [inline docs/roles/OBORoles.md:3]:
```suggestion
Each OBO Foundry Role is designed to manage a single aspect of the day-to-day activities of the OBO Foundry. 
```

File: docs/roles/OBORoles.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## General SOPs for role recruitment and retirement
+
+- The duration of an appointment to a role is 1 year, with the possibility of extension.
+- Role assignments will be reviewed during the first OBO Operations Committee call that falls in December each year.
+- Any OBO Operations Committee member may nominate a candidate for a given role. Alternatively, candidates can be self-nominated. New volunteers are particularly encouraged to participate.
+- Candidates for each role should read this document and all documents pertaining to the role of interest, acknowledge that they understand the involved duties, and agree to fulfill them to the best of their ability.
+- Each candidate must submit an application for a role to the OBO Operations Committee. The application should outline the motivation for taking on the role as well as 
+  describe the skillsets that will make them suitable for that role.
+- Applications will be reviewed by the OBO Operations Committee. Decisions for appointing a member of the community to a role will be based on vote. A positive outcome occurs (that is, the appointment is approved) if the candidate receives at least 70% in favor.
```
Comment (nlharris) [inline docs/roles/OBORoles.md:14]: I assume only current OBO Ops Committee members can vote, even though random members of the public will be allowed to sit in on Ops committee meetings.
By 70%, do you mean 70% of those who vote, or 70% of the whole ops committee? Like, if the ops committee has 20 people, and only 10 come to the meeting and vote, and 7 of them vote yes, is that sufficient? or do you need 14 yeses?

File: docs/roles/OBORoles.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- Removal from a role only impacts the privileges granted as part of the role (e.g., revoking Google Admin rights or GitHub Maintainer rights), at the discretion of the Working Group managing the role. It does not remove the person from OBO Operations Committee membership.
+ 
+
+## Expectations and responsibilities common to all roles
+
+- Role holders should announce extended absences (more than 1 month) so that the role can be covered by another OBO Foundry Operations Committee team member. 
+  Absences--or desire to retire--should be reported to the respective OBO Working Group liaison (usually the OBO Foundry Committee Technical Working Group liaison) and be announced in an email to the OBO Operations Commitee mailing list.
+- All role holders must manage themselves _pro-actively_: there is no line management structure. It is therefore critical that the role holder take their duties responsibly.
```
Comment (nlharris) [inline docs/roles/OBORoles.md:23]:
```suggestion
- All role holders must manage themselves _proactively_: there is no line management structure. It is therefore critical that the role holder take their duties responsibly. 
```

File: docs/roles/OBORoles.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+- Candidates for each role should read this document and all documents pertaining to the role of interest, acknowledge that they understand the involved duties, and agree to fulfill them to the best of their ability.
+- Each candidate must submit an application for a role to the OBO Operations Committee. The application should outline the motivation for taking on the role as well as 
+  describe the skillsets that will make them suitable for that role.
+- Applications will be reviewed by the OBO Operations Committee. Decisions for appointing a member of the community to a role will be based on vote. A positive outcome occurs (that is, the appointment is approved) if the candidate receives at least 70% in favor.
+- Any OBO Operations Committee member may suggest that an individual be removed from a role. In such case, a vote must be called to enact the removal. A postive outcomes occurs (that is, the removal is implemented) if at least 70% of the OBO Operations Committee members are in favor of removal.
```
Comment (nlharris) [inline docs/roles/OBORoles.md:15]: here you explicitly say "70% of the obo ops committee members", suggesting that you'd need 14 of 20 to vote yes, even if only 10 came to the meeting. in that case, how would the remaining votes be gathered? will votes be anonymous? what if a random non-committee member is on the call, and the vote is about someone they know?

**Review by nlharris — CHANGES_REQUESTED** [review]
looks good; a few minor comments and requests for clarification.

File: docs/roles/OBORoles.md (hunk 2 above)
Comment (nataled) [reply docs/roles/OBORoles.md:14]: We typically do those present, though if we went that route then we should probably require a minimum number of people for the vote to be valid.

File: docs/roles/OBORoles.md (hunk 4 above)
Comment (matentzn) [reply docs/roles/OBORoles.md:15]:
```suggestion
- Any OBO Operations Committee member may suggest that an individual be removed from a role. In such case, a vote must be called to enact the removal _during an OBO Operations Call_. A postive outcomes occurs (that is, the removal is implemented) if at least 70% of the OBO Operations Committee members _present in the call_ are in favor of removal.
```

Comment (matentzn) [reply docs/roles/OBORoles.md:15]: What do you think? This excluded non-OBO ops people that happen to be on a call.

File: docs/roles/OBORoles.md (hunk 2 above)
Comment (matentzn) [reply docs/roles/OBORoles.md:14]:
```suggestion
- Applications will be reviewed by the OBO Operations Committee. Decisions for appointing a member of the community to a role will be based on vote _during an OBO Operations Call_. A positive outcome occurs (that is, the appointment is approved) if the candidate receives at least 70% votes in favor by OBO Operations Committee members present during the OBO Operations Call.
```

Comment (matentzn) [reply docs/roles/OBORoles.md:14]: What about this?

File: docs/roles/OBORoles.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# OBO Foundry Roles
```
Comment (cthoyt) [inline docs/roles/OBORoles.md:1]:
```suggestion
layout: doc
title: OBO Foundry Roles
permalink: /roles/
```

**Review by nlharris — APPROVED** [review]

**Review by nataled — APPROVED** [review]

**Comment by cthoyt** [comment]
other minor follow-up on this PR, it's not linked anywhere. we can do another PR to re-organize some of the other roles information to be more coherent and better linked.

**Pull request merged**